src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationWeightedObservedDataCollectorPresenter.cs

```diff
@@ -51,14 +51,17 @@ private void addWeightedObservedDataToView(List<WeightedObservedData> allWeighte
          {
             using (new BatchUpdate(View))
             {
-               // You can get a null weighted observed data when a mapping has not been assigned observed data yet
-               allWeightedObservedData.Where(x => x != null).Each(AddObservedData);
+               allWeightedObservedData.Each(AddObservedData);
```

Comment:
AddObservedData willl nwo check for weightObservedData ==null and weightObservedData.ObservedData ==null